alwaysontop/main.js

```diff
@@ -38,27 +56,55 @@ function onAlwaysOnTopWindow(
         const win = event.newGuest = new BrowserWindow(
             Object.assign(options, {
                 backgroundColor: 'transparent',
-                width: SIZE.width,
-                height: SIZE.height,
                 minWidth: SIZE.width,
                 minHeight: SIZE.height,
-                maxWidth: SIZE.width,
-                maxHeight: SIZE.height,
                 minimizable: false,
                 maximizable: false,
-                resizable: false,
+                resizable: true,
                 alwaysOnTop: true,
                 fullscreen: false,
                 fullscreenable: false,
                 skipTaskbar: true,
                 titleBarStyle: undefined,
                 frame: false,
                 show: false
-            }, getPosition())
+            }, getPosition(), getSize())
         );
         win.once('ready-to-show', () => {
             win.showInactive();
         });
+
+        //for macOS we use the built-in setAspectRatio on resize, for other we use custom implementation
+        if (os.type() === 'Darwin') {
+            win.setAspectRatio(16/9);
+        } else {
+            win.on('will-resize', (e, newBounds) => {
+                oldSize = win.getSize();
+                const mousePos = robot.getMousePos();
+                const windowBottomRightPos = {
+                    x: newBounds.x + newBounds.width - 16,
```

Comment:
Why `-16`?

Reply:
I've set this as the diagonal resize does not necessarily start precisely on the window's right bottom corner, it has some pixels threshold on right and bottom window edges in which is still a diagonal resize